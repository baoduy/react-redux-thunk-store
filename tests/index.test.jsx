import React from "react";
import renderer from "react-test-renderer";
import Provider from "../lib";

const reducer = (store = [], action) => {
  if (action.type === "LOADDED") return [{ id: 1 }];
  return store;
};

const component = renderer.create(
  <Provider reducers={{ value: reducer }}>
    <div>Hello Store for PRD</div>
  </Provider>
);

it("can render without issues in DEV", () => {
  console.debug("NODE_ENV", process.env.NODE_ENV);
  const cs = component.toJSON();
  console.debug(cs);

  expect(cs).toMatchSnapshot();
});
