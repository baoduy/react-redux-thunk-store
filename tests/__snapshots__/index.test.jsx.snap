// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render without issues in DEV 1`] = `<div />`;
